ContentItems/Narrow selection of content items with jQuery copy.js

```diff
@@ -0,0 +1,94 @@
+/*
+This example shows how to create a filter, edit the filter by adding a new filter option to it, get content items using this filter.
+Check
+*/
+
+var _tokenID = "get token"; // See authorization examples how to get a secure token
+
+function getContentItems(portalId, searchText)
+{
+	var filterId;
+	//Create filter (use search text and portal ID)
+	$.ajax({
+		method: "POST",
+		url: "http://iprova/api/portals/content_items/filter",
+		beforeSend: function (request)
```

Comment:
// Comment: always set the accept, version and authorization for every ajax call
$.ajaxSetup({
    headers: { 'x-my-custom-header': 'some value' }
});